webrob/pages/admin/db.py

```diff
@@ -145,3 +154,50 @@ def db_remove_route(table):
     if data is not None and 'id' in data and db_find(cls, data['id']):
         db_remove(cls, data['id'])
     return jsonify(result=None)
+
+
+@app.route('/db/page/get_neem_hub_settings')
```

Comment:
the name of the route is confusing. Better call it *neem_hub_settings* or *edit_neem_hub_settings* or so